roles/splunk_common/tasks/main.yml

```diff
@@ -122,14 +122,9 @@
 - include_tasks: enable_dsp.yml
   when: "'dsp' in splunk and 'enable' in splunk.dsp and splunk.dsp.enable"
 
-- include_tasks: start_splunk.yml
-
-- include_tasks: set_certificate_prefix.yml
-  when: splunk.splunk_http_enabled|bool == true
-
-- include_tasks: clean_user_seed.yml
```

Comment:
do we need to remove this playbook from the repository? it's not used anywhere else.

Reply:
I don'r recall exactly, but I think the user seed is used to determine if this is the first time Splunk is bootstrapped or not. I remember running into a problem when doing upgrade and when using local dev build because of it

Reply:
Yeah this one I don't have strong preferences about, I couldn't find anything in the docs about what happens if we keep this around.